### PR TITLE
feat: invoice scan with standard url pattern

### PR DESCRIPTION
**Pull request by rayanfer32**
## Description

-

### Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor ( rewrite or restructure code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### Data Results

- none

### Additional Info

- none

### CODE REVIEW

File: src/components/Views/Dao/Invoices/index.jsx [hunk 1]
```diff
@@ -74,6 +74,11 @@ function InvoicesView({ username }) {
   const handleModal = (data) => {
     setModalData(data);
     setIsOpen(true);
+    window.history.replaceState(
```
Comment (shrivatsabhat) [inline src/components/Views/Dao/Invoices/index.jsx:77]: I doubt with this

File: src/components/common/PushLink/index.jsx [hunk 2]
```diff
@@ -10,14 +10,14 @@ const PushLink = ({ children, href = '', ...props }) => {
   };
 
   return (
-    <span {...props} onClick={handleClick}>
+    <a href={href} {...props} onClick={handleClick}>
```
Comment (shrivatsabhat) [inline src/components/common/PushLink/index.jsx:13]: Why it's Anchor

**Review by shrivatsabhat — CHANGES_REQUESTED** [review]

File: src/components/common/PushLink/index.jsx (hunk 2 above)
Comment (rayanfer32) [reply src/components/common/PushLink/index.jsx:13]: We want browser to detect it as url

File: src/components/Views/Dao/Invoices/index.jsx (hunk 1 above)
Comment (rayanfer32) [reply src/components/Views/Dao/Invoices/index.jsx:77]: working fine

**Review by shrivatsabhat — APPROVED** [review]

**Pull request merged**